src/emitterutils.cpp

```diff
@@ -1,6 +1,7 @@
 #include <algorithm>
 #include <iomanip>
 #include <sstream>
+#include <stdint.h>
```

Comment:
```suggestion
#include <cstdint>
```

yaml-cpp seems to use C++ includes (like MOTIS also)

Reply:
ordering is wrong with the suggestion - so please don't apply the suggestion